### DISCUSSION TITLE
Error color of terminal (vs code)

### DISCUSSION THREAD

**Discussion by DanyaSh** [dracula/dracula-theme#630] [Q&A] [answered]
What's wrong with the color of my terminal?
How to fix it?
Please! Help!
I really like this theme, but work in terminal very difficult.

![изображение](https://user-images.githubusercontent.com/10911390/134615231-72484d43-5fef-474d-bceb-b686e46da1e4.png)

**Comment by zenorocha** [comment, marked as answer]
@DanyaSh can you open an issue on https://github.com/dracula/visual-studio-code?

**Reply by DanyaSh** [reply]
ready
